testplan/web_ui/web_app.py

```diff
@@ -105,7 +105,24 @@ def get(self, report_uid, attachment_path):
                 filename=os.path.basename(attachment_path),
             )
         else:
-            raise exceptions.NotFound()
+            raise werkzeug.exceptions.NotFound()
+
+
+@_api.route("/api/v1/metadata/<string:name>/<string:section>")
+class TestplanMetadata(Resource):
```

Comment:
why not @_api.route("/api/v1/metadata/fix_spec/tags_by_num") ?

Reply:
fixed